hcloud/action.go

```diff
@@ -241,9 +198,9 @@ func (c *ActionClient) WatchOverallProgress(ctx context.Context, actions []*Acti
 				}
 			}
 
-			progress += (len(completedIDs) * 100)
+			progress += len(completedIDs) * 100
 			if progress != 0 && progress != previousProgress {
-				sendProgress(progressCh, int(progress/len(actions)))
+				sendProgress(progressCh, progress/len(actions))
```

Comment:
Why are those changes here ?

Reply:
Because they are part of the backported PR: https://github.com/hetznercloud/hcloud-go/pull/295/files#diff-8914d399a5e4a0ca8ac89a130609df3ba4006e707596cdbff04855cf929b2dcdR200-R202